core/amber/src/test/scala/edu/uci/ics/texera/unittest/workflow/operators/scan/csv/CSVScanSourceOpDescSpec.scala

```diff
@@ -100,8 +100,8 @@ class CSVScanSourceOpDescSpec extends AnyFlatSpec with BeforeAndAfter {
     csvScanSourceOpDesc.hasHeader = false
     csvScanSourceOpDesc.setContext(workflowContext)
 
-    assert(csvScanSourceOpDesc.operatorExecutor.topology.layers.length == 1)
-    assert(csvScanSourceOpDesc.operatorExecutor.topology.layers.apply(0).numWorkers == 1)
+    assert(csvScanSourceOpDesc.operatorExecutor(null).topology.layers.length == 1)
```

Comment:
Should we pass null or an OperatorSchemaInfo with empty schemas.

Reply:
Passing empty schemas will also work. I will change to that.

Reply:
Fixed by adding an empty schema for both input and output.